### ISSUE TITLE
Deprecation: int_from_bytes is deprecated

### ISSUE THREAD

**Issue by mostafa-dirac** [mitya57/secretstorage#28]
Hi!

I just upgraded [cryptography](https://pypi.org/project/cryptography/) to 3.4.1 and I got this error:

```
/usr/local/lib/python3.8/dist-packages/secretstorage/dhcrypto.py:16: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/local/lib/python3.8/dist-packages/secretstorage/util.py:25: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
```

Would you please fix this?
Thanks in advance.

**Comment by mitya57** [comment]
Merged, thanks for your PR!

**Comment by mitya57** [comment]
I have merged #27 which fixes this. I will probably make a new release in a few days, when I have more time.
